Assets/Puppet/PuppetAppCenter.cs

```diff
@@ -210,24 +186,13 @@ private IEnumerator SetEnabledCoroutine(bool enabled)
         var isPushEnabled = Push.IsEnabledAsync();
         yield return isPushEnabled;
         PushEnabled.isOn = isPushEnabled.Result;
-        var isDistributeEnabled = Distribute.IsEnabledAsync();
```

Comment:
The idea was to allow disable it without any code edits on the app side

Reply:
I'll revisit the logic and update the wiki